### ISSUE TITLE
[BUG] [Linter] block elements inside inline elements are not flagged

### ISSUE THREAD

**Issue by martinberoiz** [djlint/djLint#1387]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: mac OSX 15.6
- Python Version 3.13.5
- djLint Version 1.36.4
- template language: django

## Issue

Block elements inside inline elements (specifically a `ul` list inside a `p` tag) are not flagged as errors.
I think this should be an error.

## How To Reproduce

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="description">
    <meta name="keywords" content="testing">
    <title>Document</title>
</head>
<body>
    <p>Here comes a paragraph
        <ul>
            <li>First Item</li>
            <li>Second Item</li>
            <li>Third Item</li>
        </ul>
    </p>
</body>
</html>
```

$ djlint mytemplate.html

Linting 1/1 files ━━━━━━ 00:00

Linted 1 file, found 0 errors.

## Contents of djlint.toml/.djlintrc/pyproject.toml [tool.djlint]

{
  "profile": "django"
}
